9.2/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -223,3 +223,12 @@ function wait_for_postgresql_master() {
     sleep 1
   done
 }
+
+# this function checks where there is a script which handles migration
+# if there is script it tries to do migration
+function handle_post_init() {
+  if [ -f $POST_INIT_SCRIPT];
+  then
+    source $POST_INIT_SCRIPT
```

Comment:
Also, is there any reason why we are sourcing the script instead of executing it? It would make more sense to me